src/ram.asm

```diff
@@ -207,7 +207,17 @@ harddropBuffer: .res $14 ; $625 ; 20 bytes (!)
 linecapState: .res 1 ; $639 ; 0 if not triggered, 1 + linecapHow otherwise, reset on game init
 
 dasOnlyShiftDisabled: .res 1 ; $63A
+
+.ifdef KEYBOARD
+; Reserve 11 bytes for Family BASIC Keyboard
+    .res $3A
+newlyPressedKeys: .res 1 ; $0675
+heldKeys: .res 1 ; $0676
+keyboardInput: .res 9 ; $0677
+.else
     .res $45
+.endif
+
```

Comment:
can you rewrite this bit like this;

```asm
.ifdef KEYBOARD
    ... keyboard stuff...
.else
    .res $B
.endif

    .res $3A
```

I feel like I can "see" the ram better this way. also this way you dont need a comment to know that the keyboard uses up $B bytes of ram

Reply:
I made this change but reserved $3A first.  This keeps the 9 bytes of keyboard input on a single row aligned to the right in the memory debugger.